docs/about/faq.rst

```diff
@@ -32,15 +32,19 @@ CIViC curators and editors should express a strong interest in clinical cancer g
+A genetic variant is a non-exact copy of a biological sequence. This includes single nucleotide variants (SNVs), insertion/deletion events (indels), copy number alterations (CNV's), structural variants (SVs), transcript fusions, and other genomic/molecular events with cancer associations. The vast majority of variants described in CIViC represent mutations at the DNA level, however, alterations that manifest at the RNA, protein, or epigenetic level are also accepted so as long as there is a demonstrated clinical relevance. In general, the concept of a variant in CIViC is very flexible but it is highly preferred that the variant be something that could be measured with some quantitative assay (NGS sequencing, qPCR, etc.). Combinations of one or more variants comprise a Molecular Profile.
+
+What is a *Molecular Profile*?
+------------------------------
+There is a growing need for clinical annotation of combinations of variants from different genes, such as double hit lymphoma, which is characterized by combinations of mutations in MYC, BCL2, or BCL6, or PIK3CA mutation induced trastuzumab resistance in HER2 overexpressing breast cancer. CIViC addresses this need via use of the Molecular Profile. Molecular Profiles (MPs) are comprised of combinations of one or more variants. A simple Molecular Profile is comprised of one variant, and a complex MP is comprised of a combination of two or more variants. The addition of MPs to CIViC V2 involved an essential change  data model, in that CIViC Evidence Items (EIDs) are now associated to Molecular Profliles instead of directly to variants. This enable clinical annotation of combinations of variants which may not apply to the individual variants in isolation. Annotation of single variants is accomplished by adding Evidence Items (EIDs) simple molecular profiles (e.g. curation of clinical trial showing BRAF V600E sensitivity of vemurafenib in melanoma). Now, annotation of multi variant combinations can be accomplished by adding EIDs to complex Molecular Profiles (e.g. Osimertinib resistance in EGFR L858R and EGFR T790M and BRAF V600E positive long adenocarcinoma). Each MP in CIViC has an associated Molecular Profile page in the user interface containing a curator summary of the MP role in cancer, aliases, links to pages for the individual variants that comprise the MP, and a list of all of the curated Evidence Items which are uniquely associated to the MP. 
```

Comment:
```suggestion
There is a growing need for clinical annotation of combinations of variants from different genes, such as double hit lymphoma, which is characterized by combinations of mutations in MYC, BCL2, or BCL6, or PIK3CA mutation induced trastuzumab resistance in HER2 overexpressing breast cancer. CIViC addresses this need via use of the Molecular Profile. Molecular Profiles (MPs) are comprised of combinations of one or more variants. A simple Molecular Profile is comprised of one variant, and a complex MP is comprised of a combination of two or more variants. The addition of MPs to CIViC V2 involved an essential change to the data model, in that CIViC Evidence Items (EIDs) are now associated to Molecular Profiles instead of directly to variants. This enables clinical annotation of combinations of variants which may not apply to the individual variants in isolation. Annotation of single variants is accomplished by adding Evidence Items (EIDs) to simple molecular profiles (e.g. curation of clinical trial showing BRAF V600E sensitivity of vemurafenib in melanoma). Now, annotation of multi variant combinations can be accomplished by adding EIDs to complex Molecular Profiles (e.g. Osimertinib resistance in EGFR L858R and EGFR T790M and BRAF V600E positive long adenocarcinoma). Each MP in CIViC has an associated Molecular Profile page in the user interface containing a curator summary of the MP role in cancer, aliases, links to pages for the individual variants that comprise the MP, and a list of all of the curated Evidence Items which are uniquely associated to the MP. 
```

Reply:
I can't seem to see the change here. @tmooney

Reply:
Found it. Looks good.